### PR TITLE
[hi] Add Hindi glossary entry for "label"

### PR DESCRIPTION
**Pull request by CloudShiva7cr**
Localizes label.md in HIndi

### Description

#Fixes Localizes content/en/docs/reference/glossary/label.md in Hindi content/hi/docs/reference/glossary/label.md

### CODE REVIEW

File: content/hi/docs/reference/glossary/label.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+aka: 
+tags:
+- fundamental
+---
+ ऑब्जेक्ट्स को पहचानने योग्य विशेषताओं के साथ टैग करता है, जो यूज़र्स के लिए अर्थपूर्ण और प्रासंगिक होती हैं।
+
+<!--more--> 
+
+लेबल्स कुंजी/मान (key/value) जोड़े होते हैं जो {{< glossary_tooltip text="पॉड्स" term_id="pod" >}} जैसे ऑब्जेक्ट्स के साथ जुड़े होते हैं। इनका उपयोग ऑब्जेक्ट्स को व्यवस्थित करने और उनके सबसेट्स का चयन करने के लिए किया जाता है।
```
Comment (sftim) [inline content/hi/docs/reference/glossary/label.md:17]: How about:
```suggestion
लेबल्स कुंजी/मान (key/value) जोड़े होते हैं जो {{< glossary_tooltip text="Pod" term_id="pod" >}} जैसे ऑब्जेक्ट्स के साथ जुड़े होते हैं। इनका उपयोग ऑब्जेक्ट्स को व्यवस्थित करने और उनके सबसेट्स का चयन करने के लिए किया जाता है।
```

The API kind is Pod, and labels are definitely an API concept.

**Comment by sftim** [comment]
/retitle Add Hindi glossary entry for "label"

File: content/hi/docs/reference/glossary/label.md (hunk 1 above)
Comment (CloudShiva7cr) [reply content/hi/docs/reference/glossary/label.md]: @sftim Sure. made changes as you suggested !! Thanks!

**Review by niranjandarshann — COMMENTED** [review]
Thanks for the PR 
Changes look good to me

**Pull request merged**